ocaml/typing/ctype.ml

```diff
@@ -1675,10 +1675,10 @@ let instance_prim_layout (desc : Primitive.description) ty =
          from an outer scope *)
       if level = generic_level && try_mark_node ty then begin
         begin match get_desc ty with
-        | Tvar ({ jkind; _ } as r) when Jkind.is_any jkind ->
+        | Tvar ({ jkind; _ } as r) when Jkind.has_layout_any jkind ->
```

Comment:
Could you leave this CR around the definition of `get_jkind`?

```
(* CR layouts v2.8: This should replace only the layout component of the jkind. It's possible that we might want a primitive that accepts a mode-crossing, layout-polymorphic parameter. *)
```

Reply:
Good idea, thanks, done.